src/sagemaker/workflow/pipeline.py

```diff
@@ -61,6 +64,7 @@ def __init__(
         name: str = "",
         parameters: Optional[Sequence[Parameter]] = None,
         pipeline_experiment_config: Optional[PipelineExperimentConfig] = _DEFAULT_EXPERIMENT_CFG,
+        pipeline_definition_config: Optional[PipelineDefinitionConfig] = _DEFAULT_DEFINITION_CFG,
```

Comment:
@svia3 can we add this param as last one in the argument list, this is to avoid any backwards incpmptable changes which might break customers who use positional arguments rather name named arguments.

Reply:
Sure.